frontend/src/storybook/project.ts

```diff
@@ -51,6 +52,14 @@ export class MockProjectWrapperService
   getAvailableVisibilities(): Visibility[] {
     return Object.keys(ProjectVisibilityDescriptions) as Visibility[];
   }
+
+  getProjectTypeDescription(type: ProjectType): string {
```

Comment:
```suggestion
  get projectTypeDescription(type: ProjectType): string {
```

You can use getter for it and the next function.